wagtail/wagtailadmin/tests/test_pages_views.py

```diff
@@ -1470,6 +1470,7 @@ def test_preview_on_edit(self):
         self.assertTemplateUsed(response, 'tests/simple_page.html')
         self.assertContains(response, "I&#39;ve been edited!")
 
+    @override_settings(ALLOWED_HOSTS=['testserver', 'childpage.example.com'])
```

Comment:
If you need `testserver` in there, another option is `@modify_settings(ALLOWED_SETTINGS={'append': 'childpage.example.com'})`, otherwise looks fine.